sqldb/sqlite_js.go

```diff
@@ -0,0 +1,18 @@
+//go:build js || (windows && (arm || 386)) || (linux && (ppc64 || mips || mipsle || mips64))
+
+package sqldb
+
+import "fmt"
+
+// SqliteStore is a database store implementation that uses a sqlite backend.
+type SqliteStore struct {
```

Comment:
Does this actually work out of the box with `wasm`? Would assuming the underlying syscalls get in the way?

Reply:
No it's not supported unfortunately. We just aim for compilation success.